src/services/codefixes/disableJsDiagnostics.ts

```diff
@@ -35,15 +35,21 @@ namespace ts.codefix {
                 fixId: undefined,
             }];
         },
-        fixIds: [fixId], // No point applying as a group, doing it once will fix all errors

```

Comment:
This comment is better written a few lines up -- no point applying the `// @ts-nocheck` fix as a group. There *is* a point applying `// @ts-ignore` as a group.